test/langtools/jdk/javadoc/testJavadocTester/TestJavadocTester.java

```diff
@@ -430,8 +431,8 @@ private String fix(String item) {
      * @param items the strings
      */
     private String[] fix(String... items) {
-        return List.of(items).stream()
+        return Stream.of(items)
```

Comment:
While List.of(items).stream() is _not_ equivalent to Stream.of(items), that `fix` method (as a whole) still behaves the same way. One difference between those two approaches of creating an ordered stream is their behavior in regard to the null item.